src/plugins/url-change.js

```diff
@@ -7,6 +7,10 @@ import { useNavStore } from '~/stores/nav'
  * Normally, the `src/middleware/embed.js` middleware does this on every
  * route change. However, it does not run on the initial render.
  * So, for the initial render this plugin runs when router is ready.
+ *
+ * @param root0
```

Comment:
Is it possible to remove these changes in the `.js` files? I guess we are going to move to TS in them soon, so it seems like it adds more work to rename `root0` annotations, and then to replace them with TS

Reply:
If we kept param documentation in the TS docstrings then we'd need these anyway, FWIW. It uses `root0` as the default name (and `root1` and so on if there are multiple destructured arguments), ideally we'd replace these with more sensible names.